api/app/Generators/PoolCandidateCsvGenerator.php

```diff
@@ -162,38 +169,45 @@ public function generate(): self
                 });
                 $values[] = implode(', ', $userSkills->toArray());
 
+                $poolQuestionIds = $candidate->pool->generalQuestions->pluck('id')->toArray();
```

Comment:
Can we call this $poolGeneralQuestionIds to differentiate it from $poolScreeningQuestionIds on L184?

Reply:
Great suggestion. I also updated the other reference in [clearer distinction between questions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11526/commits/5d27b85d10f35c792665556fb9872db4f68b430f)